src/main/features/claim/form/models/claimAmountRow.ts

```diff
@@ -38,13 +38,27 @@ export class ClaimAmountRow {
     return new ClaimAmountRow(undefined, undefined)
   }
 
+  static translateToNumber (input?: any): any {
+    const numberValue = _.toNumber(input)
+    const isExponential = numberValue.toString().match(new RegExp('\\d*[Ee][+-]?\\d*?'))
```

Comment:
Did the UX say we need this? What sane person would use scientific notation when issuing a money claim?

Reply:
citizen frontend had amount restricted to 10,000 however user still can enter number more than 13 digit, when it get changed to exponential. and our validation has to handle exponential number and it looses it's value during validation process